website/docs/components/tag.mdx

```diff
@@ -9,7 +9,7 @@ import ComponentLinks from "../../src/components/component-links"
 
 # Tag
 
-Tag component is used for items that need to be labeled, categorized, or
+Tag component is used for items that needs to be labeled, categorized, or
```

Comment:
```suggestion
Tag component is used for items that need to be labeled, categorized, or
```

"need" is correct here because "items" is plural :)

Reply:
Right right !!, I will change it.